src/__tests__/dsl.test.ts

```diff
@@ -0,0 +1,55 @@
+import { describe, expect, test } from '@jest/globals';
+import { parseDSL, productions } from '../dsl';
+
+describe('parseDSL', () => {
+    test('parses AND query', () => {
+        let sx = parseDSL('tcp.port eq 23 and tcp.port eq 445');
+        expect(sx.lexErrors).toStrictEqual([]);
+        expect(sx.parseErrors).toStrictEqual([]);
```

Comment:
you could use `.toHaveLength(0)` here but not sure if it makes any difference

Reply:
sounds good. Done.